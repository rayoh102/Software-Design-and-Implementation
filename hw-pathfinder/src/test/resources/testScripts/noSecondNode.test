# Find a path with Dijkstra's algorithm when
# the second node doesn't exist

CreateGraph simpleGraph

AddNode simpleGraph A

FindPath simpleGraph A B
