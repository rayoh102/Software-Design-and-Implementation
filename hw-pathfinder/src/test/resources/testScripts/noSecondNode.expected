# Find a path with Dijkstra's algorithm when
# the second node doesn't exist

created graph simpleGraph

added node A to simpleGraph

unknown node B